src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,28 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+
+
+/**
+ * Lists last shown persons in the address book to the user sorted by name in alphabetical order.
+ */
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Displays last-shown persons in the address book sorted by name in alphabetical order as a list with index numbers.\n"
+            + "Example: " + COMMAND_WORD;
+
+
+    @Override
+    public CommandResult execute() {
+        List<ReadOnlyPerson> relevantPersonsCopy = new ArrayList<>(this.relevantPersons);
+        relevantPersonsCopy.sort(Comparator.comparing(person -> person.getName().toString()));
```

Comment:
Do we want to do a case-sensitive sort?

Reply:
Also, the rest of the codebase uses `.fullName` to access the name as a string rather than `.toString()`. If we want to be consistent I believe we should stick to `.fullName`. This makes sense as we are using these objects like plain ol data structures.

We can potentially use a separate PR to change all such objects to use accessor methods instead, if needed.

Reply:
Good idea! Updated sort to be case-insensitive and use `fullName` instead of `toString()`